src/js/components/uploadDetachedFiles/UploadDetachedFilesPage.jsx

```diff
@@ -360,16 +421,32 @@ export default class UploadDetachedFilesPage extends React.Component {
 
 		let validationBox = null;
 		if(this.state.showValidationBox) {
+			uploadFilesBox = null;
+			datePicker = null;
+			subTierAgencyIcon = null;
+			contentSize = 'col-xs-12 mt-60 mb-60';
+			agencyClass = null;
+			title = null;
+
 			const type = {
 				fileTitle: 'File D2: Award',
 				fileTemplateName: 'detached_award.csv',
 				requestName: 'detached_award',
 				progress: '0'
 			}
-			validationBox = <ValidateDataFileContainer type={type} data={this.state.jobResults} />;
+			validationBox = <ValidateDataFileContainer type={type} data={this.state.jobResults}/>;
 			if(!this.state.headerErrors && this.state.validationFinished) {
 				validationBox = <ValidateValuesFileContainer type={type} data={this.state.jobResults} />;
 			}
+			if(this.state.headerErrors && this.state.validationFinished) {
+				// validationButton = <button className='pull-right col-xs-3 us-da-button' onClick={this.uploadFile.bind(this)}>Re-Upload</button>;
+			}
+			else if(!this.state.headerErrors && this.state.validationFinished && this.state.submit) {
```

Comment:
This can probably be refactored to be a bit simpler (i.e. check `!this.state.headerErrors && this.state.validationFinished` and then move this and the next call inside to check `this.state.submit`)